src/utils.ts

```diff
@@ -0,0 +1,698 @@
+// Dependencies
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+import Fuse from 'fuse.js';
+import ErrorWithoutStack from './error-without-stack';
+
+// Interface declarations
+interface InputObject {
+	command: string;
+	data?: string | number;
+	[propName: string]: boolean | string | number | undefined;
+}
+
+// Interface declarations
+export interface AppSettings {
+	name: string | null;
+	packageName: string | null;
+	version: string | null;
+	//[propName: string]: string | number | null;
```

Comment:
I was not sure whether the intention is to allow the 'AppSettings' to have additional arbitrary properties or not. Since this object is retrievable, I thought it conceivable that additional properties could be useful to clients. What say you?

Reply:
No, only settings that are supported by Waterfall CLI should be used.